recipes-devtools/cmake/cmake_%.bbappend

```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
Use _class-nativesdk here as well and shouldn't this be BPN?

Reply:
yes this should be BPN